src/color.ts

```diff
@@ -137,10 +138,10 @@ export function stringToRGBA(color: string): number[] {
     const {prefix, value} = splitBracket(color);
 
     if (!prefix || !value) {
-      return;
+      return undefined as never;
     }
     const arr = splitComma(value);
-    const colorArr: number[] = [];
+    const colorArr = [] as any as [number, number, number, number];
```

Comment:
@xiaoxiangmoe 

thank you 👍 

I think better.
```js
const colorArr:  [number, number, number, number] = [0, 0, 0 1];
```

Reply:
I have a question:

* should we use readonly tuple in function return type?

Reply:
@xiaoxiangmoe


It's Good.